run_model.py

```diff
@@ -26,21 +27,34 @@ def main():
     is_flag=True,
     help="Only runs the county summary if true.",
 )
-def run_county(state=None, deploy=False, summary_only=False):
+@click.option(
+    '--git-hash',
+    type=str,
+    help='''
+    | Git hash of the commit in covid-data-public to use.
+    | If provided, covid-data-public must have no pending changes.
+    | If omitted, the repository will be used as-is'''
+)
+def run_county(state=None, deploy=False, summary_only=False, git_hash=None):
     """Run county level model."""
-    min_date = datetime.datetime(2020, 3, 7)
-    max_date = datetime.datetime(2020, 7, 6)
+    with data_version(git_hash) as version:
```

Comment:
I don't love having to wrap all code that uses the version in a context manager - I think this would be a good chance for a decorator to enclose this context manager. 

I gave this approach a shot (#106 ) - let me know what you think!

Reply:
Merged and very lightly updated.